src/mongocxx/uri.hpp

```diff
@@ -48,11 +51,15 @@ class MONGOCXX_API uri {
     static const std::string k_default_uri;
 
     ///
-    /// Constructs a uri from an optional MongoDB uri string. If no uri string is specified,
-    /// uses the default uri string, 'mongodb://localhost:27017'.
+    /// Constructs a uri from an optional MongoDB URI string. If no URI string is specified,
```

Comment:
```suggestion
    /// Constructs a URI from an optional MongoDB URI string. If no URI string is specified,
```

Reply:
I took the view that `uri` refers to the class, which is lowercase, and `URI` is the concept.